### PR TITLE
BREAKING: migrate to typescript

### PR DESCRIPTION
**Pull request by saenyakorn**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -26,13 +26,18 @@
...
     "autoprefixer": "^9.6.0",
     "cssnano": "^4.1.10",
     "husky": "^7.0.2",
     "lint-staged": "^11.2.0",
     "prettier": "^2.4.1",
-    "tailwindcss": "^1.0.3"
+    "tailwindcss": "^1.0.3",
+    "tsparticles": "^1.36.0",
+    "typescript": "^4.4.4",
+    "yarn": "^1.22.15"
```
Comment (new5558) [inline package.json]: Why we have yarn here?

File: src/components/sections/members.tsx [hunk 2]
```diff
@@ -37,14 +45,36 @@ const Profile = ({ avatarURL, profileURL, username, imageAlt }) => (
...
 const Members = () => {
-  const [members, setMembers] = useState([])
+  const [members, setMembers] = useState<GithubMemberProps[]>([])
   useEffect(() => {
     ;(async function fetchAPI() {
       for (let page = 0; page < maxPage; page++) {
-        const data = await (
-          await fetch(`https://api.github.com/orgs/thinc-org/public_members?per_page=${maxPerPage}&page=${page}`)
-        ).json()
+        const { data }: { data: GithubMemberProps[] } = await axios.get(
```
Comment (new5558) [inline src/components/sections/members.tsx]: Should we handle error when fetch fail?

**Review by new5558 — COMMENTED** [review]
Can we try to deploy on personal Vercel before merge this?

File: package.json [hunk 3]
```diff
@@ -26,13 +26,18 @@
   "devDependencies": {
     "@fullhuman/postcss-purgecss": "^1.2.0",
     "@trivago/prettier-plugin-sort-imports": "^2.0.4",
+    "@types/react": "^17.0.30",
     "@zeit/next-css": "^1.0.1",
+    "add": "^2.0.6",
```
Comment (new5558) [inline package.json]: Why need this?

File: package.json (hunk 1 above)
Comment (saenyakorn) [reply package.json]: WTF

File: package.json (hunk 3 above)
Comment (saenyakorn) [reply package.json]: WTF

**Comment by saenyakorn** [comment]
> Can we try to deploy on personal Vercel before merge this?

Sure

File: src/components/sections/members.tsx (hunk 2 above)
Comment (saenyakorn) [reply src/components/sections/members.tsx]: OK

**Review by new5558 — APPROVED** [review]
LGTM

**Pull request merged**